### PR TITLE
ZCS-15319: Exiting command execution after completion

### PR DESCRIPTION
**Pull request by zimsuchitgupta**
**Problem**
zmplayredo utility is not exiting after execution

**Solution**
Command was not getting terminated after successful execution so added `System.exit(0)`.

### CODE REVIEW

**Review by preshitaagrawal — APPROVED** [review]

File: store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java [hunk 1]
```diff
@@ -375,6 +375,7 @@ public static void main(String[] cmdlineargs) throws Throwable {
             player.playback();
         } finally {
             teardown();
+            System.exit(0);
```
Comment (dasiyogesh) [inline store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java:378]: why is this needed now?

Comment (ronstra-synacor) [reply store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java:378]: Yeah, does this actually stop it from hanging? It seemed like the hangup was something in the teardown(), that calls Zimbra.shutdown(), which just shuts down a lot of threads for the server, but if there was a problem, it's still not going to make it to the next System.exit(0) statement, right?

Comment (dasiyogesh) [reply store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java:378]: there seems to be something going in that shutdown, Suchit is checking on it.

Comment (log2akshat) [reply store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java:378]: The above solution is not very convincing. Perhaps a thread dump analysis can uncover the mystery.

Try filtering out the stack trace for the state of the threads during `Zimbra.shutdown()`, which, as @ronstra-synacor mentioned, is responsible for shutting down many server threads. Focus primarily on differentiating between `RUNNABLE` and `BLOCKED` threads.

Comment (dasiyogesh) [reply store/src/java/com/zimbra/cs/redolog/util/PlaybackUtil.java:378]: Some new threads which are not exiting are causing the issue which will be refactored and fixed in the respective projects. In the mean time we can go with this change for the tool which has no functional impact.

**Review by dasiyogesh — APPROVED** [review]

**Comment by zimsuchitgupta** [comment]
Closing this PR as this will be taken care of in another ticket.
